src/components/Masters.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="masters">
-    <h4>Select Masters</h4>
+    <h4 style="font-family:'Trebuchet MS', sans-serif;">Select Masters</h4>
```

Comment:
why are we adding fonts in HTML section and not adding them in CSS section?

Reply:
ok will fix it !!